s2spy/rgdr/label_alignment.py

```diff
@@ -7,6 +8,7 @@
 from typing import List
 from typing import Set
 from typing import Tuple
+
```

Comment:
Where did the empty lines between the imports come from? AFAIK our isort configuration does not do that.

See also the other .py files

Reply:
Thanks for the solution. Indeed I was confused at the beginning. But after some googling I found some codes actually have these empty lines. Thought I might be wrong before. It is nice that we can remove it by changing the setting. Looks much better now.